src/cgutils.cpp

```diff
@@ -171,6 +171,31 @@ static Value *stringConstPtr(IRBuilder<> &irbuilder, const std::string &txt)
     }
 }
 
+// --- MDNode     ---
+Metadata *to_md_tree(jl_value_t *val) {
+    if (val == jl_nothing)
+        return NULL;
+    Metadata *MD;
+    if (jl_is_symbol(val)) {
+        MD = MDString::get(jl_LLVMContext, jl_symbol_name((jl_sym_t*)val));
+    } else if jl_is_bool(val) {
```

Comment:
Missing parentheses around the condition here (and below)?
```suggestion
    } else if (jl_is_bool(val)) {
```